core/fetcher/graffiti.go

```diff
@@ -0,0 +1,121 @@
+// Copyright © 2022-2025 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package fetcher
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	eth2api "github.com/attestantio/go-eth2-client/api"
+
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/eth2wrap"
+	"github.com/obolnetwork/charon/app/version"
+	"github.com/obolnetwork/charon/core"
+)
+
+const obolToken = "OB"
+
+func ClientGraffitiMappings() map[string]string {
+	return map[string]string{
+		"Teku":       "TK",
```

Comment:
```suggestion
		"teku":       "TK",
```

https://github.com/Consensys/teku/blob/2823ba95f767a0a54e7cb0071c45ded6fc331665/data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/node/Version.java#L27